.github/workflows/unit-tests.yml

```diff
@@ -100,15 +100,15 @@ jobs:
 
       - name: Check code formatting
         run: |
-          bash ./tests/run_code_style.sh install
-          bash ./tests/run_code_style.sh lint
+          pip install -r requirements-check.txt
+          pre-commit run -a
 
       - name: Run Mypy
         # https://github.com/pytorch/ignite/pull/2780
         #
         if: ${{ matrix.os == 'ubuntu-latest' }}
         run: |
-          bash ./tests/run_code_style.sh mypy
+          mypy
```

Comment:
Should we specify mypy.ini here: https://github.com/pytorch/ignite/blob/master/mypy.ini

Reply:
Not needed as mypy will automatically look for that file.